deployment/jenkins/jobs/Digital Mktplace Library/config.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version='1.1' encoding='UTF-8'?>
```

Comment:
should we rename this file?

Reply:
It'll all have to be done manually using jenkins UI, I can try to genericize it, but it won't matter, and I may mess up some config accidentally